src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/AgencyLocationEstablishmentRepository.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.repository;
+
+
+import org.springframework.data.repository.CrudRepository;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.AgencyLocationEstablishment;
+
+public interface AgencyLocationEstablishmentRepository extends CrudRepository<AgencyLocationEstablishment, AgencyLocationEstablishment.Pk> {
```

Comment:
I couldn't see where this was being used (maybe I should review in Intellij)

Reply:
Yeah I probably take this it out.  It is a hangover from the first commit...